openhtf/__init__.py

```diff
@@ -146,6 +146,7 @@ def Stop(self, *_):
         _LOG.error('Test state: %s', self._executor.GetState())
         self._executor.Stop()
         self._executor = None
+    raise KeyboardInterrupt
```

Comment:
nit: iirc, style prefers raising instances rather than the exception type like this?

similarly a nit, can you add a comment here as to why you need to raise this, the python docs are actually a bit cryptic, but it looks as though this signal handler gets called in the main thread, between other instructions to the global interpreter?